desktopView.css

```diff
@@ -0,0 +1,112 @@
+@media screen and (max-width: 768px) {
+  nav {
+    display: flex;
+    justify-content: space-around;
+    background-color: white;
+    align-items: center;
+    padding: 1rem;
+  }
+
+  .navbar__ul {
+    display: flex;
+    align-items: center;
+    height: 20px;
+  }
+
+  .logo {
+    width: 76px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #6070ff;
+    text-decoration: none;
+  }
+
+  nav img {
+    display: none;
+  }
+
+  .navrbar__link {
+    margin-left: 23px;
+    list-style: none;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-size: 15px;
+    font-weight: 500;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    color: #344563;
+  }
+
+  .description {
+    background-color: white;
+    background-image: url(img/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    border-bottom-left-radius: 6rem;
+    padding-bottom: 6rem;
+  }
+
+  .header {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    padding: 90px 100px 2px 100px;
+    font-weight: 700;
+    line-height: 50px;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    font-size: 30px;
+    text-align: left;
+  }
+
+  .parg {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 28px;
+    color: #344563;
+    padding: 15px 100px 2px 100px;
+    font-size: 17px;
+    letter-spacing: 0;
+    text-align: left;
+  }
+
+  .let__go {
+    padding: 1.2rem 100px 0 100px;
+    color: #396df2;
+  }
+
+  .social__media {
+    display: flex;
+    padding: 12px 100px 0 100px;
+  }
+
+  .social__media__link {
+    list-style: none;
+    margin-right: 1rem;
+  }
+
+  .social__media__img {
+    width: 13px;
+    color: #505f79;
+  }
+
+  /* ===================POST===================== */
+
+  .post__div {
+    display: flex;
```

Comment:
- [ ] Kindly, make user you stack to the design (margins, colors, font-size,... ) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.) 😃. 

#### Figma design
![Screenshot 2023-07-17 164849](https://github.com/sibomana123/MV__Portfolio/assets/57408419/a38c70cd-1fb0-4476-bf53-10232ba11ebb)

#### Your design
![Screenshot 2023-07-17 165148](https://github.com/sibomana123/MV__Portfolio/assets/57408419/d159c7b0-46a3-4e5a-ad6e-46bc7500866e)